tests/bin/polykey.test.ts

```diff
@@ -1,11 +1,69 @@
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+import readline from 'readline';
+import Logger, { LogLevel, StreamHandler } from '@matrixai/logger';
 import * as execUtils from '../utils/exec';
 import { runTestIfPlatforms } from '../utils';
 
 describe('polykey', () => {
-  runTestIfPlatforms('lunix', 'docker')('default help display', async () => {
+  runTestIfPlatforms('linux', 'docker')('default help display', async () => {
     const result = await execUtils.pkStdio([]);
     expect(result.exitCode).toBe(0);
     expect(result.stdout).toBe('');
     expect(result.stderr.length > 0).toBe(true);
   });
+  runTestIfPlatforms('docker')('format option affects STDERR', async () => {
```

Comment:
Same here, does this mean this test **only** runs under docker platform?

Reply:
I got a little confused by the wording but that test runs both locally and in the testing phase of the CI/CD so I think it means if the platform is `docker` then run it (but if the platform isn't specified also run it).

Reply:
Ok I think the wording needs to be changed in a commit in staging. It should be a strict `AND` logic and OR logic between conditions.

This is why it shouldn't have changed to just a string list. A Boolean conditional is more obvious and flexible.

Reply:
It runs be default if the platform isn't specified. This is because we need to run all tests during normal testing and I'd rather not have to set `PK_TEST_PLATFORM` for that. I think we can make it non-implicit by using `runTestIfPlatforms(undefined, 'docker')`?